sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClientOptions.cs

```diff
@@ -29,6 +29,7 @@ public CommunicationIdentityClientOptions(ServiceVersion version = LatestVersion
                 ServiceVersion.V2022_06_01 => "2022-06-01",
                 ServiceVersion.V2022_10_01 => "2022-10-01",
                 ServiceVersion.V2023_10_01 => "2023-10-01",
+                ServiceVersion.V2025_04_01_PREVIEW => "2025-04-01-preview",
```

Comment:
should we update everywhere the version to `2025-03-02-preview` as discussed during the meeting that we will ship 3 features all together?